Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob/Definitions/Source.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+#pragma warning disable 1591
+namespace Frends.AzureBlobStorage.ReadBlob
+{
+    public class Source
+    {
+        /// <summary>
+        ///     Use either Uri+SAS Token or Connection string
+        /// </summary>
+        [DefaultValue("https://xx.blob.xx.xx.net/")]
+        [DisplayFormat(DataFormatString = "Text")]
+        public string Uri { get; set; }
+
+        [DisplayFormat(DataFormatString = "Text")]
+        public string SasToken { get; set; }
```

Comment:
SAS token is a secret, so you should change it to secret using `[PasswordPropertyText]`.